src/components/AddressHistory.js

```diff
@@ -10,8 +10,15 @@ import dateFormatter from '../utils/date';
 import hathorLib from '@hathor/wallet-lib';
 import PropTypes from 'prop-types';
 import PaginationURL from '../utils/pagination';
-import helpers from '../utils/helpers';
 import SortableTable from './SortableTable';
+import { connect } from "react-redux";
+
+
+function mapStateToProps(state) {
+  return {
+    decimalPlaces: state.serverInfo?.decimal_places ?? hathorLib.constants.DECIMAL_PLACES,
```

Comment:
This `??` operation is done everywhere. Can we start the `serverInfo` state with `{ decimalPlaces: hathorLib.constants.DECIMAL_PLACES }`, so we don't need to do it every time?

Reply:
Changed to treat the null/undefined/missing cases on the redux instead of the components.